projects/1.16.1/assets/sophisticatedbackpacks/sophisticatedbackpacks/lang/zh_cn.json

```
@@ -0,0 +1,124 @@
+    "item.sophisticatedbackpacks.backpack.tooltip.shift": "左 Shift",
+    "item.sophisticatedbackpacks.backpack.tooltip.upgrades": "升级",
+    "item.sophisticatedbackpacks.backpack.tooltip.inventory": "物品栏",
+    "item.sophisticatedbackpacks.backpack.tooltip.empty": "无升级卡或物品",
+    "item.sophisticatedbackpacks.iron_backpack": "铁制背包",
+    "item.sophisticatedbackpacks.gold_backpack": "金制背包",
+    "item.sophisticatedbackpacks.diamond_backpack": "钻石背包",
+    "item.sophisticatedbackpacks.upgrade_base": "升级卡模板",
+    "item.sophisticatedbackpacks.pickup_upgrade": "拾取升级",
+    "item.sophisticatedbackpacks.pickup_upgrade.tooltip": "使背包捡起物品",
+    "item.sophisticatedbackpacks.advanced_pickup_upgrade": "高级拾取升级",
+    "item.sophisticatedbackpacks.advanced_pickup_upgrade.tooltip": "使背包捡起物品\n有更多的筛选项",
+    "item.sophisticatedbackpacks.filter_upgrade": "过滤升级",
+    "item.sophisticatedbackpacks.filter_upgrade.tooltip": "过滤输入/输出背包的物品",
+    "item.sophisticatedbackpacks.advanced_filter_upgrade": "高级过滤升级",
+    "item.sophisticatedbackpacks.advanced_filter_upgrade.tooltip": "过滤输入/输出背包的物品\n有更多的筛选项",
+    "item.sophisticatedbackpacks.magnet_upgrade": "磁铁升级",
+    "item.sophisticatedbackpacks.magnet_upgrade.tooltip": "在一定范围内将物品吸入背包",
+    "item.sophisticatedbackpacks.advanced_magnet_upgrade": "高级磁铁升级",
+    "item.sophisticatedbackpacks.advanced_magnet_upgrade.tooltip": "在更大的范围内将物品吸入背包\n有更多的筛选项",
+    "item.sophisticatedbackpacks.feeding_upgrade": "自动喂食升级",
+    "item.sophisticatedbackpacks.feeding_upgrade.tooltip": "用背包里的食物喂玩家",
+    "item.sophisticatedbackpacks.compacting_upgrade": "压缩升级",
+    "item.sophisticatedbackpacks.compacting_upgrade.tooltip": "将物品压缩为对应的压缩变体\n仅 2x2 配方",
+    "item.sophisticatedbackpacks.advanced_compacting_upgrade": "高级压缩升级",
+    "item.sophisticatedbackpacks.advanced_compacting_upgrade.tooltip": "将物品压缩为对应的压缩变体\n包括 2x2，3x3 和更多的筛选项",
+    "item.sophisticatedbackpacks.void_upgrade": "虚空升级",
+    "item.sophisticatedbackpacks.void_upgrade.tooltip": "清空过滤器中选定的项目",
+    "item.sophisticatedbackpacks.advanced_void_upgrade": "高级虚空升级",
+    "item.sophisticatedbackpacks.advanced_void_upgrade.tooltip": "清空过滤器中选定的项目\n有更多的筛选项",
+    "item.sophisticatedbackpacks.restock_upgrade": "取货升级",
+    "item.sophisticatedbackpacks.restock_upgrade.tooltip": "手持背包 Shift 右击容器从中存入物品",
+    "item.sophisticatedbackpacks.advanced_restock_upgrade": "高级取货升级",
+    "item.sophisticatedbackpacks.advanced_restock_upgrade.tooltip": "手持背包 Shift 右击容器从中取出物品\n有更多的筛选项",
+    "item.sophisticatedbackpacks.deposit_upgrade": "卸货升级",
+    "item.sophisticatedbackpacks.deposit_upgrade.tooltip": "手持背包 Shift 右击容器向内放置物品",
+    "item.sophisticatedbackpacks.advanced_deposit_upgrade": "高级卸货升级",
+    "item.sophisticatedbackpacks.advanced_deposit_upgrade.tooltip": "手持背包 Shift 右击容器向内放置物品\n有更多的筛选项",
+    "item.sophisticatedbackpacks.refill_upgrade": "补充升级",
+    "item.sophisticatedbackpacks.refill_upgrade.tooltip": "保持补充玩家物品栏中所选物品数量",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "item.sophisticatedbackpacks.refill_upgrade.tooltip": "不断地补充玩家物品栏中所选物品的数量",
```